### ISSUE TITLE
Union Type error on Slices

### ISSUE THREAD

**Issue by p-tul** [prismicio/prismic-gatsby#487]
### Versions

- gatsby-source-prismic: v5.2.3
- gatsby: v4.3.0
- node: v17.2.0

### What is actually happening?

I have updated the 'gatsby-source-prismic' plugin from v1.0.2 to v5.2.3. When running `yarn start`, the build fails during the extracting-queries step.

```

 ERROR 

Missing onError handler for invocation 'extracting-queries', error was 'Error: Union type PrismicDevelopmentUpdatesDataBody1SlicesType must define one or more member types.'.
Stacktrace was 'Error: Union type PrismicDevelopmentUpdatesDataBody1SlicesType must define one or more member types.
    at assertValidSchema (/Users/p-tul/Documents/eFront/yarrabend/node_modules/gatsby/node_modules/graphql/type/validate.js:69:11)
    at validate (/Users/p-tul/Documents/eFront/yarrabend/node_modules/gatsby/node_modules/graphql/validation/validate.js:51:35)
    at extractOperations (/Users/p-tul/Documents/eFront/yarrabend/node_modules/gatsby/src/query/query-compiler.js:209:20)
    at processQueries (/Users/p-tul/Documents/eFront/yarrabend/node_modules/gatsby/src/query/query-compiler.js:167:45)
    at compile (/Users/p-tul/Documents/eFront/yarrabend/node_modules/gatsby/src/query/query-compiler.js:78:19)
    at updateStateAndRunQueries (/Users/p-tul/Documents/eFront/yarrabend/node_modules/gatsby/src/query/query-watcher.ts:223:40)
    at extractQueries (/Users/p-tul/Documents/eFront/yarrabend/node_modules/gatsby/src/services/extract-queries.ts:18:3)'
```

---
gatsby-config.js
```
{
	resolve: 'gatsby-source-prismic',
	options: {
		repositoryName: process.env.GATSBY_PRISMIC_REPO_NAME,
		accessToken: process.env.PRISMIC_ACCESS_TOKEN,
		customTypesApiToken: process.env.PRISMIC_CUSTOM_TYPES_API_TOKEN,
		linkResolver: (doc) => linkResolver(doc),
	},
},
```

---

I have even gone so far as to remove all graphql queries from the repo with no change to the result so it must be happening withing 'gatsby-source-prismic`.

**Comment by angeloashmore** [comment]
Hi @p-tul, this happens because your "Development Updates" Custom Type contains a Slice Zone with no Slices. I will update the plugin to fix this issue.

For an immediate fix, you can remove the empty Slice Zone in the "Development Updates" Custom Type. It may be located in a secondary tab (note that the type contains `Body1`, which means there's at least one other Slice Zone in the document already).

**Comment by angeloashmore** [comment]
@p-tul The fix has been published in `gatsby-source-prismic` `v5.2.4`. Could you give it a try and let me know if it fixes your issue? Thank you!
